src/Projection/PdoEventStoreReadModelProjector.php

```diff
@@ -595,8 +596,11 @@ private function createProjection(): void
 EOT;
 
         $statement = $this->connection->prepare($sql);
-        // we ignore any occuring error here (duplicate projection)
-        $statement->execute([$this->name, $this->status->getValue()]);
+        try {
+            $statement->execute([$this->name, $this->status->getValue()]);
+        } catch (PDOException $e) {
```

Comment:
Ok, sounds good, but don't catch exception then, check the error code on the statement object instead.

About the second PR, I will take a look this weekend.